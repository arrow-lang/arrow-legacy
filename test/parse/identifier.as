index;
index = 430;
index + 40;
ようこそ + index // 23;
