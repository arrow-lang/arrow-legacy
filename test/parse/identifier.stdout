<?xml version="1.0" encoding="utf-8"?>
<Module>
 <Identifier>index</Identifier>
 <Assign>
  <Identifier>index</Identifier>
  <Integer base="10">430</Integer>
 </Assign>
 <Add>
  <Identifier>index</Identifier>
  <Integer base="10">40</Integer>
 </Add>
 <Add>
  <Identifier>ようこそ</Identifier>
  <IntDiv>
   <Identifier>index</Identifier>
   <Integer base="10">23</Integer>
  </IntDiv>
 </Add>
</Module>
